third-party/chpl-venv/venv-use-sys-python.py

```diff
@@ -4,32 +4,176 @@
 
 import os
 import sys
+import re
+
+# This fixes up the chpl-venv installation in a Cray Chapel module that was built using a non-standard
+# Python installation (ie, the Python used was not installed under "/usr/...").
+#
+# 1. Remove the python executables from the virtualenv.
+#    This effectively forces the virtualenv scripts to use the system python.
+# 2. Rewrite symlinks in the virtualenv that point to the non-standard Python installation
+# 3. Rewrite shebang lines (in scripts) in the virtualenv that point to the non-standard Python exe.
+
+# Without this, "start_test" will not work if the Cray Chapel module is installed on a different
+# machine (ie, if no Python is installed in the same location as when the module was built).
+
+# As noted, these fixes are only applicable when building a Cray Chapel module under certain conditions.
+# This has been tested only for the cray-xc platform, where the Cray Chapel module was:
+#  - built on a cray-xc CLE 5 (system Python version 2.6, non-standard user-installed Python 2.7), and
+#  - installed on a cray-xc CLE 6 (system Python version 2.7).
+
+
+def rm_python_bins( bindir ):
+    # This removes any file objects named "python", "python2", etc found in bindir.
+
+    # bindir is the abs path to ./third-party/chpl-venv/$(platform)/$(python_version)/chpl-virtualenv/bin
+
+    if os.path.isdir( bindir ) and not os.path.islink( bindir ):
+        for f in os.listdir( bindir ):
+            abs_f = os.path.join( bindir, f )
+            # abs_f is the abs path to some file system object under bindir
+            if f in [ 'python', 'python2', 'python2.6', 'python2.7' ]:
```

Comment:
This would be more general, unless there are certain python versions that we *don't* want to delete.

```python
for f in os.listdir(bindir):
    if f.startswith('python'):
        abs_f = os.path.join(bindir, f)
        print('Removing {0}'.format(abs_f))
        os.remove(abs_f)
```